checkout/views.py

```diff
@@ -0,0 +1,190 @@
+from django.shortcuts import render, redirect, reverse
+from django.conf import settings
+from django.contrib import messages
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+
+from .forms import OrderForm
+from .models import Order, OrderLineItem
+from products.models import Product
+from profiles.models import UserProfile
+from profiles.forms import UserProfileForm
+
+
+def checkout(request):
+
+	if request.method == 'POST':
+		cart = request.session.get('cart', {})
+
+		form_data = {
+			'full_name': request.POST['full_name'],
```

Comment:
I suppose I'm missing something - why are you not using the regular approach of instantiating the form object with `request.POST`?
With your current approach a missing field would raise an exception

Reply:
I believe it's because of the extra POST param save-info in the template (a boolean that determines whether they want to save the information to their profile or not), which isn't in the order form: 

```
{% if user.is_authenticated %}
<input class="form-check-input ml-2 mr-0" type="checkbox" id="id-save-info" name="save-info" checked>
{% endif %}
```

I think using the standard method threw an exception trying to find a field for it in the model. I'll review this and make sure it's solid. What do you mean a missing field would raise an exception? Take a look at the checkout.html template and let me know if you still see an issue.